ts/components/LoadingSpinnerOverlay.tsx

```diff
@@ -57,6 +57,7 @@ class LoadingSpinnerOverlay extends React.Component<Props> {
                     onPress={onCancel}
                     cancel={true}
                     style={styles.cancelButtonStyle}
+                    testID={"loadingSpinnerOverlayCancelButton"}
```

Comment:
I occasionally see test IDs in the form of PascalCase "NameComponentTestID" and other times "nameComponent" as camelCase. Is there a standard rule for selecting a testId's name?

Reply:
I think there is not 😅 Would you like me to change the ones defined in this PR to a specific one?